docker/docker-compose.yml

```diff
@@ -0,0 +1,49 @@
+version: "3"
+services:
+  bitcoin-core:
+    container_name: bitcoin-core
+    image: ruimarinho/bitcoin-core:0.19.1-alpine
+    command:
+      -printtoconsole
+      -testnet=1
+      -txindex=1
+      -server=1
+    environment:
+      - BITCOIN_DATA=/data
+    volumes:
+      - ${DATA_VOL}:/data
+  bitcoin-ion:
+    container_name: bitcoin-ion
+    image: node:10.20.1-alpine3.11
+    command: "npm run bitcoin"
+    working_dir: /ion
+    links:
+      - "bitcoin-core"
+      - "mongo"
```

Comment:
Sanity: Is there a need to specify `links`? From [links documentation](https://docs.docker.com/compose/compose-file/#links) it appears that usage is not necessary. Are you using it to enforce dependency start up sequence? Also it says `link` is legacy and should be avoided if possible.

Reply:
Link is used here to allow name resolution for dependent containers.  I am not 100% sure that depends_on will give you this without setting up a specific network.  Basically this allows the internal network and name resolution without the need to create a network.